src/transformers/training_args.py

```diff
@@ -247,7 +247,7 @@ class TrainingArguments:
 
     def __post_init__(self):
         if self.disable_tqdm is None:
-            self.disable_tqdm = logger.getEffectiveLevel() > logging.WARN
+            self.disable_tqdm = logging.get_verbosity() <= logging.INFO
```

Comment:
I don't understand this change. Is the order different from `logging`? Also I'm pretty sure we want the progress bars at the info level.

Reply:
Ah yes, this was a mistake. Reverting.